ush/eva/marine_gdas_plots.yaml

```diff
@@ -0,0 +1,99 @@
+# template YAML file to create EVA YAML files
+# based on obs spaces listed in JEDI YAML files
+diagnostics:
+- data:
+    type: IodaObsSpace
+    datasets:
+      - name: experiment
+        filenames:
+          - @FILENAME@
+        @CHANNELSKEY@
+        groups:
+          - name: ObsValue
+            variables: &variables @VARIABLES@
+          - name: ObsError
+          - name: ombg
+          - name: oman
+          - name: hofx0
+          - name: EffectiveQC0
+          - name: MetaData
+          - name: PreQC
+  transforms:
+
+    # bkg 
+    - transform: arithmetic
+      new name: experiment::bkg::${variable}
+      equals: experiment::ObsValue::${variable}-experiment::ombg::${variable}
+      for:
+        variable: *variables
+
+    # Generate omb that passed QC for JEDI
+    - transform: accept where
+      new name: experiment::ombgPassedQC::${variable}
+      starting field: experiment::ombg::${variable}
+      where:
+        - experiment::PreQC::${variable} == 0
+      for:
+        variable: *variables
+
+    # Generate oma that passed QC for JEDI
+    - transform: accept where
+      new name: experiment::omanPassedQC::${variable}
+      starting field: experiment::oman::${variable}
+      where:
+        - experiment::PreQC::${variable} == 0
+      for:
+        variable: *variables
+
+    # Generate bkg that passed QC for JEDI
+    - transform: accept where
+      new name: experiment::bkgPassedQC::${variable}
+      starting field: experiment::bkg::${variable}
+      where:
+        - experiment::PreQC::${variable} == 0
+      for:
+        variable: *variables
+
+
+  graphics:
+
+    # ---------- Map Plots ----------
+    # Map plot of ObsValue 
+    # --------
+
+    - batch figure:
+        variables: *variables
+        @CHANNELSKEY@
+      dynamic options:
+        - type: vminvmaxcmap
+          data variable: experiment::@GRPNAME@::${variable}
```

Comment:
I guess I don't see why you would be doing this versus utilizing the same infrastructure that `${variable}` is using to loop from within EVA, can you give an example/use case?

Reply:
@GRPNAME@ is for the group of the data we intend to use to produce the figure. For example, if group is ObsValue, then only the data under ObsValue will be plotted; if the group is bkg, then the data under bkg will be plotted and so on. We assume all the groups contains the same variable but with different data.